ajax/libs/emoji-picker/package.json

```diff
@@ -24,7 +24,7 @@
   ],
   "autoupdate": {
     "source": "git",
-    "target": "git://github.com/one-signal/emoji-picker.git",
+    "target": "git://github.com/onesignal/emoji-picker.git",
```

Comment:
I think this should be `git://github.com/OneSignal/emoji-picker` ?

Reply:
Agreed with @PeterDaveHello